consensus/pow/worker.go

```diff
@@ -0,0 +1,10 @@
+package consensus
+
```

Comment:
move up to consensus folder for interface

Reply:
just keep it.